content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```diff
@@ -1,41 +1,78 @@
+In this topic, you will learn how to test and verify AWS GameLift Gem feature integrations on your local machine by using [Amazon GameLift Anywhere](https://docs.aws.amazon.com/gamelift/latest/developerguide/fleets-creating-anywhere.html). 
 
-GameLift Local is a command line tool that starts a self-contained version of the managed Amazon GameLift service. GameLift Local also provides a running event log of server process initialization, health checks, and API calls and responses. GameLift Local recognizes a subset of the AWS SDK actions for GameLift. You can make calls from the AWS CLI or from your game client. All API actions perform locally, just as they do in the GameLift web service. For more information, refer to [Testing Your Integration](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-testing-local.html#integration-testing-local-start) in the Amazon GameLift Developer Guide.
+GameLift Anywhere can be used to integrate hardware from your environment into your Amazon GameLift game hosting. Amazon GameLift Anywhere registers your hardware with Amazon GameLift in an Anywhere fleet; you can iteratively test and build your game server project using your own hardware.
 
-{{< caution >}}  
-GameLift Local does not currently support testing of FlexMatch features.
-{{< /caution >}}
-
-## 1. Start GameLift Local
-
-With GameLift Local, you can verify the following: (Refer to [Set Up GameLift Local](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-testing-local.html#integration-testing-local-start))
+With GameLift Anywhere, you can verify the following:
 
 *   Your game server is correctly integrated with the Server SDK and is properly communicating with the GameLift service to start new game sessions, accept new players, and report health and status.
     
 *   Your game client is correctly integrated with the AWS SDK for GameLift and is able to retrieve information on existing game sessions, start new game sessions, join players to games and connect to the game session.
     
 
-After downloading GameLift Local, you should be able to start it by (please sure you are passing the same port while starting game, as GameLift client endpoint needs to use the same port)
+## 1. Create a Location
+
+Create a location for your custom resources. A custom location labels the location of your hardware that Amazon GameLift uses to run your games in Anywhere fleets.
+When creating your custom location, the location name must start with `custom-`.
+
+```sh
+aws gamelift create-location --location-name custom-location-1 --region <Region>
+```
+
 
-```shell
+## 2. Create an Anywhere Fleet
 
-$ java -jar GameLiftLocal.jar -p 9080
+Creating an Anywhere fleet is a much faster process compared to creating a regular AWS fleet, which usually takes about an hour to setup.
 
+```sh
+aws gamelift create-fleet --name AnywhereFleet --compute-type ANYWHERE --locations Location=custom-location-1 --region <Region>
```

Comment:
Add a note to save the Fleet-Id for the next steps (maybe provide an example of what a fleet id looks like)